ui/src/utils/prop-types.js

```diff
@@ -126,6 +126,7 @@ export const taskPriority = oneOf([
   'LOW',
   'VERY_LOW',
   'LOWEST',
+  'normal',
```

Comment:
Ah, we are indeed missing the normal priority of a task. The convention in graphql is to capitalize constants then in web-server we can map the capital letters to their actual values.

Reply:
So what I should do is capitalize on the `normal` constant?, It's not piking it when it's normal. So it seems it's picking data that is already from the db and the taskPriority is in small letters.

Reply:
The code to make it work is on the diff that I posted on my review. Also, whenever we change things in `web-server`, we currently need to restart the process to see the changes. Does that help?

Reply:
Yes that really helped. Thanks @helfi92